examples/scenarios/syslog.md

```diff
@@ -0,0 +1,40 @@
+# Syslog
+
+Once you have Stanza installed and running from the [quickstart guide](./README.md#quick-start), you can follow these steps to configure Syslog monitoring.
+
+## Prerequisites
+
+It may be necessary to add an inbound firewall rule on Windows. To do this:
+ * Navigate to Windows Firewall Advanced Settings, and then Inbound Rules
+ * Create a new rule and set the Rule Type to "Port"
+ * For Protocol and Ports, select "UDP" and a specific local port of 514
+ * For Action, select "Allow the connection"
+ * For Profile, apply to "Domain", "Private", and "Public"
+ * Set a name to easily identify rule, such as "Allow Syslog Inbound Connections to 514 UDP"
+
+## Configuration
+
+This is an example config file that can be used in the Stanza install directory. The Syslog plugin supports UDP and TCP logs, using UDP by default.
+
+```yaml
+pipeline:
+  # To see the Syslog plugin, go to: https://github.com/observIQ/stanza-plugins/blob/master/plugins/syslog.yaml
+  - type: syslog
+    listen_port: 514
+    listen_ip: "0.0.0.0"
+    connection_type: udp
+    protocol: rfc5424
```

Comment:
This one is annoying, but the value is wrong. I wish it was simpler, but it looks like this: `rfc5424 (IETF)`
```
...
  - name: protocol
    label: Protocol
    description: The protocol of received syslog messages (`rfc3164` or `rfc5424`)
    type: enum
    valid_values:
      - rfc3164 (BSD)
      - rfc5424 (IETF)
...
```

Reply:
Corrected